### PR TITLE
new style idea

### PR DESCRIPTION
**Pull request by mshd**
How about it?

### CODE REVIEW

File: src/components/Node/Node.scss [hunk 1]
```diff
@@ -2,9 +2,9 @@
   box-sizing: border-box;
   position: absolute;
   transition: all ease-in-out 0.2s;
-  background-color: #eee;
+  background-color: rgb(250,238,222);
   border: 1px solid lightgrey;
-  border-radius: 5px;
+  border-radius: 3px;
```
Comment (mshd) [inline src/components/Node/Node.scss:7]: I guess this can be changed back to 5px

**Comment by ogroppo** [comment]
I need to pull this branch and have a closer look, as I cannot figure out just from the code

**Comment by ogroppo** [comment]
> Merge branch 'master' of https://github.com/ogroppo/wikiforest

You still haven't updated the remote url?

File: src/components/Node/Node.js [hunk 2]
```diff
@@ -109,6 +113,39 @@ export default function Node({
...
+            )}
+            {hairColor && (
+              <span
+                className="hairColor"
+                title={hairColor.itemLabel}
+                style={{
+                  color:  "#"+hairColor.hex
+                }}
+              >
+            <GiBeard />
```
Comment (ogroppo) [inline src/components/Node/Node.js:144]: mmmh, doesn't convince me much

File: src/components/Node/Node.scss [hunk 3]
```diff
@@ -2,9 +2,9 @@
   box-sizing: border-box;
   position: absolute;
   transition: all ease-in-out 0.2s;
-  background-color: #eee;
+  background-color: rgb(250,238,222);
```
Comment (ogroppo) [inline src/components/Node/Node.scss:5]: not sure about this, at the minute, still taking the stance of using colour where meaning needs to be conveyed.

File: src/components/Node/Node.scss (hunk 1 above)
Comment (ogroppo) [reply src/components/Node/Node.scss:7]: I didn't mind to have to img full size actually

File: src/components/Node/Node.scss [hunk 4]
```diff
@@ -67,7 +68,7 @@
       display: inline;
       border: 0;
       padding: 0;
-      font-size: 13px;
+      font-size: 14px;
```
Comment (ogroppo) [inline src/components/Node/Node.scss:71]: mmh

**Review by ogroppo — CHANGES_REQUESTED** [review]
Sorry man, I know there's quite some work here but my gut feeling about this eye and hair thing is not a good one

**Review by ogroppo — APPROVED** [review]

**Pull request merged**